roles/lib_openshift/src/class/oc_obj.py

```diff
@@ -50,7 +50,11 @@ def create(self, files=None, content=None):
         if files:
             return self._create(files[0])
 
-        content['data'] = yaml.dump(content['data'])
+        if hasattr(yaml, 'RoundTripDumper'):
+            content['data'] = yaml.dump(content['data'], Dumper=yaml.RoundTripDumper)
```

Comment:
The conditional is for safety for the cases when at the top ruamel.yaml is not loaded, thus RoundTripDumper is not present. We have these types of checks in the libraries, this one was missing it. 

The real change is that we had to use the dumper here because from rumael.yaml-0.13 onwards a regular dump won't produce a yaml that the yaml parser in openshift will understand. It is possible that down the line origin will update gopkg.in/yaml.v2 and this will be redundant, but until then we need to make sure openshift can read yamls we create.